scoresheets/Receptionist.tex

```diff
@@ -1,22 +1,30 @@
 \begin{scorelist}[startbutton=false,timelimit=5]
 	\scoreheading{Main Goal}
-	\scoreitem[2]{15}{Guide the guest to the other guests (navigate to the guest group)}
-	\scoreitem[2]{50}{Look in the direction of navigation or at the navigation goal}
-	\scoreitem[2]{50}{Introduce a new guest to every other guest and the host}
+	\scoreitem[2]{30}{show the guest around (navigate to the beverage area and living room)}
+	\scoreitem[1]{30}{Look in the direction of navigation or at the navigation goal}
+	\scoreitem[2]{20}{identify favorite drink}
+	\scoreitem[2]{20}{identify interest of guest}
```

Comment:
what does `identify` mean? Is it fine for the robot to just ask for a hobby and listen for an answer?